span-normalizer/src/test/resources/configs/span-normalizer/test-application.conf

```diff
@@ -4,37 +4,24 @@ service.admin.port = 8099
 main.class = org.hypertrace.core.spannormalizer.SpanNormalizer
 
 span.type = jaeger
-
-flink.job {
-  parallelism = 4
-  metrics {
-    metrics.reporters = "prometheus"
-    metrics.reporter.prometheus.class = "org.hypertrace.core.serviceframework.metrics.flink.PrometheusReporter"
-  }
-}
-
-flink.source {
-  type = kafka
-  topic = jaeger-spans
-  kafka {
-    bootstrap.servers = "localhost:9092"
-    group.id = "jaeger-spans-to-raw-spans-job"
-    auto.offset.reset = "latest"
-    enable.auto.commit = true
-    auto.commit.interval.ms = 5000
-  }
+input.topic = "jaeger-spans"
+output.topic = "raw-spans-from-jaeger-spans"
+
+kafka.streams.config = {
+  application.id = jaeger-spans-to-raw-spans-job
```

Comment:
call this as `span-normalizer`

Reply:
+1

Reply:
@buchi-busireddy while this is correct, I believe providing an explanation on the why would help everyone to understand the change.